ethcontract-generate/Cargo.toml

```diff
@@ -20,3 +20,4 @@ proc-macro2 = "1.0"
 quote = "1.0"
 syn = "1.0.12"
 url = "2.1"
+web3 = { version = "0.15", default-features = false }
```

Comment:
I think this should be re-exported from `ethcontracts-common` and then reused from there (kind of how its done for `Address`). My main motivation for that is that it:
- means there are less instances of `web3` versions that need to be updated
- it in principle decouples the `web3` dependency from `ethcontract-generate` (you can imagine that `H256` is swapped for an alternate implementation).

Reply:
Agree. Changed it to rexport TransactionHash.